NewForce-Cohort-7/tabloidfullstack-bloodyscallwags
# Ticket #3 View Posts

Description
[Trello Ticket #3](https://trello.com/c/eAuL4Irh/3-3-view-all-posts) - Added necessary files and folders to give users ability to view all approved posts sorted by their publication date.

Testing Instructions

git add and git commit whatever you're working on on your branch

git checkout main

git fetch --all

git checkout

If you don't see the most recent version of my files, git pull origin

If you haven't created the Tabloid MVC database, please do so.

Use this to fill out tables and columns: [SQL data](https://github.com/NewForce-Cohort-7/tabloidfullstack-bloodyscallwags/blob/main/SQL/01_Tabloid_Create_DB.sql)

Use this to seed the database: [SQL Seed Data](https://github.com/NewForce-Cohort-7/tabloidfullstack-bloodyscallwags/blob/main/SQL/02_Tabloid_Seed_Data.sql)

Click the execute with debugger solid green button to launch the app

In browser, please go to Login on the nav bar and log in with this email: [foo@bar.com](mailto:foo@bar.com)

This should redirect you back to the home page.

You should be able to see Posts in the Nav Bar logged in as this user. Click the link in the Nav Bar. This should redirect you to a list of all *approved* posts that include the Author, Title, Category, icons for later use, and is sorted by the publication date. 






Checklist:
[x] My code follows the style guidelines of this project
[x] I have performed a self-review of my code
[x] I have commented my code, particularly in hard-to-understand areas
[x] My changes generate no new warnings or errors
[x] I have added test instructions that prove my fix is effective or that my feature works
